docs/admin/access.rst

```diff
@@ -4,32 +4,32 @@ Access control
 ==============
 
 Weblate comes with a fine-grained privilege system to assign user permissions
-for the whole instance, or in a limited scope.
+for the whole instance with predefined roles, or by assigning one or more
+groups of permissions to users for everything, or individual projects, components,
```

Comment:
This needs to be a cursory overview.
Otherwise the whole thing is a riddle where the reader has to establish the connections.